cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,36 @@
-describe('', () => {
-  before(() => {
+const { generateAticle } = require('../support/genArticle');
 
+describe('test for article flow', () => {
+  
+  beforeEach(() => {
+    cy.register().then(({username,email,password}) => {
```

Comment:
when you register, you receive the token which you can set as a cookie. Do not need to register and than login via api